.travis.yml

```diff
@@ -46,6 +46,19 @@ matrix:
         - DESC="findbugs and pmd"
         - CMD="mvn clean compile pmd:check findbugs:check"
 
+    # checkstyle injection compile no error (oraclejdk8)
+    - jdk: oraclejdk8
+      env:
+        - DESC="checkstyle injection compile no error"
+        - CMD1="git clone https://github.com/checkstyle/checkstyle &&"
+        - CMD2="cp src/main/resources/com/github/checkstyle/regression/extract/ExtractInfoGeneratorTest.java "
+        - CMD3="checkstyle/src/test/java/com/puppycrawl/tools/checkstyle/ExtractInfoGeneratorTest.java &&"
+        - CMD4="cp src/main/resources/com/github/checkstyle/regression/extract/Json.java "
+        - CMD5="checkstyle/src/test/java/com/puppycrawl/tools/checkstyle/Json.java &&"
+        - CMD6="cd checkstyle &&"
+        - CMD7="mvn clean compile"
+        - CMD="$CMD1$CMD2$CMD3$CMD4$CMD5$CMD6$CMD7"
```

Comment:
Maybe better in #50 ? The test on generated file needs a lot of process of read/convert/... And many logic are included in #50 . Should we combine the changes of these two PR and close one of them?